test/Makefile

```diff
@@ -93,33 +145,15 @@ endif
 all: default
 default: build
 stop: clean
-build: agbot-image exchange-db-image exchange-image css-mongo-db-image hashicorp-vault-image
-build-remote: agbot-image
+build: test-image hashicorp-vault-image
+build-remote: test-image
 run: test
 distclean: realclean
 
-DOCKER_AGBOT_CNAME := agbot
-DOCKER_EXCH ?= http://exchange-api:8080/v1
-CSS_URL ?= https://css-api:9443
-VAULT_ADDR = http://vault:8200
-API_KEY ?= 0
-CERT_LOC ?= 1
-DOCKER_EXCH_CNAME = exchange-api
-DOCKER_EXCHDB_CNAME := exchange-db
-DOCKER_REG_CNAME := e2edevregistry
-DOCKER_REG_PW_CNAME := htregistry
-DOCKER_REG_USER := testuser
-DOCKER_REG_PW := testpassword
-DOCKER_CSS_CNAME := css-api
-DOCKER_CSSDB_CNAME := css-db
-DOCKER_HASHICORP_VAULT_CNAME := vault
-MUL_AGENTS ?= 0
-
-run-agbot: agbot-docker-prereqs test-network
-	touch /tmp/horizon
-	touch /tmp/css.crt
-	@echo "Handling $(DOCKER_AGBOT_CNAME)"
-	docker/docker_run.bash "$(DOCKER_AGBOT_CNAME)" \
+run-test: test-image e2edevtest-docker-prereqs test-network
```

Comment:
This comment can be addressed in a follow-on PR: It would be good for the make targets that actually run the tests (and cleanup) be converted to scripts. You need to keep in mind that now many people that aren't core anax developers will be running e2edev tests. Those people will probably be using off-the-shelf versions of most of the containers, and only be building the 1 container they are working on (e.g. exchange), which they may build themselves.

Reply:
ok, I will put it in the issue 2232 so that we can work on it later.